src/components/MainContent/TaskModel.tsx

```diff
@@ -24,52 +25,63 @@ const renderDualTextOnlyStems = (stems: Stem[]) =>
   renderStemsBy('Appropriate Stems for Dual-Text Stimuli', stems);
 
 const renderFormatExample = (example: string, sectionName: string) => {
+  let elements: JSX.Element | undefined;
+
   // If example data has 'NA' or 'Examples', ignore them temporarily.
   // If API changes how to present none for example, this block should change.
   if (example === 'NA') return;
   if (example === 'Examples') return;
 
-  return (
+  elements = (
     <Section name={sectionName}>
       <SubHeader text={'Format Example'} />
       <Passage>{parseContent(example)}</Passage>
     </Section>
   );
+
+  return elements;
 };
 
-export const TaskModel = ({ task, stems, evidences, scoringRule, index }: TaskModelProps) => {
+export const TaskModel = ({
+  task,
+  stems,
+  evidences,
+  scoringRule,
+  index,
+  names
+}: TaskModelProps) => {
   const taskPrefix = `taskModel${index}`;
   const desc = parseContent(task.taskDesc);
 
```

Comment:
Lets destructure all of the variabled we're using in `task` and `names`

Reply:
I refactored the code to use destructuring and there we're no naming conflicts so we'll be fine in this file.